### ISSUE TITLE
problem with load image/textures

### ISSUE THREAD

**Issue by huitre39** [cflavio/yorg#9]
i downloaded the master version but i have many problem with file/link image.

for example program call /asset/images/gui/cursor.dds
but .dds file not exist, i have cursor.png

i have same problem with /asset/images/icons/facebook.png or all social network. all icons in this folder not exist ! (i have only psd file)

thanks for advance.

**Comment by cflavio** [comment]
Hi! As I wrote in the [readme](https://github.com/cflavio/yorg/blob/master/README.md), you should run _scons images=1_ for creating image files (moreover, I wrote that building is supported only for Linux). If you are not on Linux you can download a built version from [here](http://www.ya2tech.it/download.html) (e.g. the _source_ build) and grab assets from here. I am sorry but I can't maintain building for other OSes (I hope that someone will port it on other OSes in the future). Thank you very much and I apologize for the issue!

**Comment by huitre39** [comment]
ok thanks for your help.

But why can not include directly all the files (.dds, png...etc) in github ?

**Comment by cflavio** [comment]
The files that aren't included are files that are built from "source" files. As instance, _dds_ files are built from _png_/_jpg_/_psd_ files, _bam_ files are built from _egg_ files, ... Typically you don't want to include built files in a repo - it is a _source_ repo - e.g. you are not going to include executable and binary files built from source files. If I included dds and bam files, this repo would be huge (the folder with the built assets is almost 1GB now). Anyway, I am distributing the built files (in the link I copied in my previous message), so everyone can grab them without polluting this repo. :)
